src/entities/alerts/index.ts

```diff
@@ -0,0 +1,45 @@
+/** @format */
+import { GlobalConfig } from "../../config";
+import Network, { Models } from "../../network";
+import { IPagedEntity } from "../../models";
+
+import { IAlert, IAlertSummary } from "./models";
+
+export class Alerts {
+  baseUrl: string = "alerts";
+
+  /**
+   * Retrieves an alert matching the given identifier
+   * @param identifier - Id related to a specific alert
+   * @returns A single alert or null if not found;
+   */
+  public async get(identifier: string): Promise<IAlert | null> {
+    let urlSegments = [GlobalConfig.planIdentifier, this.baseUrl, identifier];
+
+    try {
+      const url = Network.buildApiUrl(urlSegments);
+      const res = await Network.get<IAlert>(url);
+      return res;
+    } catch (err) {
+      return null;
```

Comment:
I wonder if we should add in some config option to allow these network errors to be thrown/sent to Raygun? 
This looks like it would swallow errors & I'm sure some customers might want to know when a network error occurs

Reply:
I'll come back to this in another PR as I may change the networking.